.github/workflows/setlist-parser.yml

```
@@ -0,0 +1,217 @@
+            done
+          else
+            echo "削除するsetlistファイルがありません"
+          fi
+
+          # 変更があるかチェック
+          if git diff --staged --quiet; then
+            echo "コミットする変更がありません"
+          else
+            echo "変更をコミット中..."
+            git commit -m "chore: update assets and clean up processed setlist files
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)